events.html

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
This seems fine, but it's just pure HTML, right ?

Reply:
Pure, thrice-distilled HTML.